LICENSE

```diff
@@ -1,21 +0,0 @@
-MIT License
-
-Copyright (c) 2018 NikitaGordia
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
```

Comment:
THe LICENSE file is required. Please add this again

Reply:
Added